File: java/Desafio2.java
import java.util.Scanner;

public class Desafio2 {

    public static void main(String[] args){
        Scanner scanner = new Scanner(System.in);

        System.out.print("Informe a altura do triangulo em cm: ");
        double altura = scanner.nextDouble();

        System.out.print("Informe a base do triangulo em cm: ");
        double base = scanner.nextDouble();

        double area = Desafio2.calculaArea(base, altura);
        Desafio2.impressao(area);

    }

    private static double calculaArea(double base, double altura){
        return (base * altura) / 2;
    }

    private static void impressao(double area){
        System.out.println("A área é de " + area + " centímetros");
    }
}

File: python/Desafio2.py
class Desafio2:
    altura = float(input("Informe a altura do triangulo: "))
    base = float(input("Informe a base do triangulo: "))

    @classmethod
    def calculaArea(cls):
        return cls.base * cls.altura / 2

    @classmethod
    def imprime(cls):
        area = cls.calculaArea()
        print(f"A área do triangulo é de {area} centimetros")


Desafio2.imprime()
